lib/msal-common/src/request/AuthorizationCodeUrlRequest.ts

```diff
@@ -3,8 +3,11 @@
  * Licensed under the MIT License.
  */
 
+import { Prompt, ResponseMode } from "../utils/Constants";
+
 /**
- * @type AuthorizationCodeUrlRequest: Request object passed by user to retrieve a Code from the server (first leg of authorization code grant flow)
+ * @type AuthorizationCodeUrlRequest: Request object passed by user to retrieve a Code from the
+ * server (first leg of authorization code grant flow)
```

Comment:
Thanks for this. Did we verify typedocs format we would need for comments like this?

Reply:
Have not verified. If we need a different format, I'll update as part of the PR for enabling typedoc.